### PR TITLE
8.0 allow delete and permission fix

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

**Review by thomaspaulb — APPROVED** [review]

File: mail_edit/models/mail_message.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from openerp import api, models, SUPERUSER_ID, _
+from openerp.exceptions import except_orm
+
+
+class MailMessage(models.Model):
+    _inherit = 'mail.message'
+
+    @api.multi
+    def check_access_rule(self, operation):
+        if operation in ['write', 'unlink']:
```
Comment (hbrunn) [inline mail_edit/models/mail_message.py:13]: @daramousk this is a big problem: What happens when operation is none of those two? You just made all mail messages readable to everyone

Comment (hbrunn) [reply mail_edit/models/mail_message.py:13]: ping @thomaspaulb

Comment (hbrunn) [reply mail_edit/models/mail_message.py:13]: I think you should remove all of this, and add an error handler on the javascript side

**Comment by thomaspaulb** [comment]
@daramousk Merged this prematurely, I undid it but the PR status remains merged. Maybe you can open a new PR.

**Pull request merged**